src/trading/TakeOrder.sol

```diff
@@ -0,0 +1,393 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/trading/ITakeOrder.sol';
+import 'ROOT/Controlled.sol';
+import 'ROOT/libraries/ReentrancyGuard.sol';
+import 'ROOT/libraries/math/SafeMathUint256.sol';
+import 'ROOT/libraries/math/SafeMathInt256.sol';
```

Comment:
Is this being used?  I think you're only using SafeMathUint256.

Reply:
Fixed.